core/src/test/scala/unit/kafka/message/MessageWriterTest.scala

```diff
@@ -34,7 +34,7 @@ class MessageWriterTest extends JUnitSuite {
 
   private def mkMessageWithWriter(key: Array[Byte] = null, bytes: Array[Byte], codec: CompressionCodec): Message = {
     val writer = new MessageWriter(100)
-    writer.write(key = key, codec = codec) { output =>
+    writer.write(key = key, codec = codec, timestamp = Message.NoTimestamp, timestampType = CreateTime, magicValue = Message.MagicValue_V0) { output =>
```

Comment:
Should we use message v1? Since v1 is the default message format, it seems that v1 should be the format that we use in most tests.